src/App.js

```diff
@@ -8,9 +8,7 @@ import './App.css';
 export default function App() {
   return (
     <Router>
-
       <Navigation />
-      <h1>BOOKSTORE📖</h1>
       <Routes>
         <Route path="/" element={<Books />} />
```

Comment:
-  to match the [required design](https://pedantic-beaver-cb923b.netlify.app/#/) please fix this issues 🙏  : 
- [ ] Add some padding/margins for the page content
- [ ] update the navbar links' color & remove the default text-decorations
- [ ] [OPTIONAL] I highly recommend that you use the "Montserrat",sans-serif" font to match the design

![image](https://user-images.githubusercontent.com/89282221/215289638-a1ef0195-b104-4b0c-a45e-9a93e79e8227.png)